Document CAN support for stm32f072b_disco and nucleo_l432kc

erwango opened the pull request:
CAN support was not correctly documented.

Fixes #12052

erwango committed:
boards: stm32072b_disco: Document CAN support

Document CAN support for board stm32072b_disco.

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

dbkinder commented on boards/arm/nucleo_l432kc/doc/nucleol432kc.rst:
```diff
@@ -116,6 +116,10 @@ The Zephyr nucleo_l432kc board configuration supports the following hardware fea
+| CAN       | on-chip    | can                                 |
++-----------+------------+-------------------------------------+
+
+.. note:: CAN feature requires additional CAN transceiver
```
The word ``additional`` implies there is a transceiver already there and one more is needed.  Is that what you intended?  If not, then this might need to be written as:
```
.. note:: CAN feature requires a CAN transceiver.
```
and the same change in the other .rst file.

erwango replied:
ok, done. Txs for the review!

erwango committed:
boards: nucleo_l432kc: Document CAN support

Document CAN support for board nucleo_l432kc and
add can to board yaml "support" section.

Fixes #12052

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

dbkinder reviewed (approved):
LGTM, thanks!
